src/tests/mod.rs

```diff
@@ -133,19 +151,23 @@ fn execute_piranha_and_check_result(
 /// * test_name: Name of the test (identifier)
 /// * relative_path: relative path such that `test-resources/<language>/<relative_path>` leads to a directory containing the folders `input` and `configurations`
 /// * expected_number_of_matches: expression returning the expected number of matches
+/// * matches_frequency: The expected frequency of the matches
```

Comment:
```suggestion
/// * matches_frequency: The expected frequency for each of the matches
```

To make things more clear.

Also, do we still need `expected_number_of_matches` or is that just computable from some sort of sum of `matches_frequency`?

Edit: Looking below, I think this does make the `expected_number_of_matches` argument obsolete and we should remove it.